packages/core/test/types.test.ts

```diff
@@ -472,7 +496,7 @@ describe('events', () => {
           type: 'FOO';
         }
       },
-      entry: raise('FOO')
+      entry: raise<any, any, any>('FOO')
```

Comment:
Can we get this to work with just `entry: raise('FOO')`? We shouldn't expect to write `<any, any, any>`

Reply:
It works without `any`s - it's just now what this test is about. The test title is this:
> should not use actions as possible inference sites 1

This test is specifically testing that we end up inferring `{ type: 'FOO' }` for the `TEvent` despite those `any`s here